components/dashboard/src/Insights.tsx

```diff
@@ -71,7 +75,11 @@ export const Insights = () => {
 
                 {errorMessage && (
                     <Alert type="error" className="mt-4">
-                        {errorMessage instanceof Error ? errorMessage.message : "An error occurred."}
+                        {isLackingPermissions
+                            ? "You don't have permission to access this organization's insights."
```

Comment:
```suggestion
                            ? "You don't have `owner` permission to access this organization's insights."
```

Reply:
Mentioning the role makes this information for tangible and actionable.

Reply:
Sounds great! Gave it a bit more formatting pizzaz to match other places in the dashboard and pushed as [`b0f3011` (#20593)](https://github.com/gitpod-io/gitpod/pull/20593/commits/b0f3011136d34a31bd9f97e949970f4d09597b5f).

![Uploading image.png…]()